New Craftsman Page

lpalma opened the pull request:
### Not ready for merge

Continuing the branch created by @mashooq, this PR:

- [x] Replace the `Apply` buttons on the Careers page with links to the Apprentice and Craftsman pages;
- [x] Add the Application Form buttons to Apprentice / Craftsman pages;
- [x] Add the proper content to the Apprentice page;

The Application Form used in this PR is the same as the currently live version. The new Application Form will be implemented in a future iteration.

mashooq committed:
updating craftsmanpage and adding translation

lpalma committed:
add button description to yaml file

lpalma committed:
resolve merge conflicts

MiyamotoAkira committed:
Merge branch 'master' into new_craftsman_page

MiyamotoAkira committed:
Added apply form to individual pages

The apply forms that were present on the default careers form have been added to the individual crafstman and apprentice pages. Also, apprentice had the full page replaced with a copy of the crafstman one until we get the new version text

cyplo commented on careers/become_an_apprentice.html:
```diff
@@ -1,145 +1,194 @@
 ---
 layout: default
-title: Become an Apprentice
+title: Become a Apprentice
```
I think `an` was warranted here, what do you think ?

MiyamotoAkira committed:
New data on Apprentice page

We have reverted to a similar version as it was before. We have added data provided by David. There is an specifiction section on the en.yml file for it. Still needs to be added to the es.yml file

MiyamotoAkira committed:
Merge branch 'master' into new_craftsman_page

lpalma committed:
add partial for 'life of apprentice' articles

lpalma committed:
replace hard-coded blocks with included partials

MiyamotoAkira committed:
Fixed issue with background color on benefits

MiyamotoAkira committed:
Added interested button at the end.

Copied from the version at the top. I have changed the indentation, so is left aligned correctly on the document

MiyamotoAkira committed:
Changes on apprenticeship data

A few unused entries have been deleted. The structure has been copied to the es version, but still with the english language

MiyamotoAkira committed:
Added enrolment button at the bottom

MiyamotoAkira committed:
Replicated structure of benefits

We have replicated the same structure of benefits on the apprentice page as they are on the crafstman page. Because the benefits are nearly identical, we have just added the mentor benefit on the en.yml file and moved benefits outside of become_a_craftsman

MiyamotoAkira committed:
Added benefits to spanish yaml

For completion purposes, I have copied the same structure of benefits from the english yaml page into the spanish yaml page

MiyamotoAkira committed:
Added option to serve spanish version

When trying to work on the become_an_apprentice page we were getting an error. `rake serve` only creates the english version. `rake es` creates a spanish version but with a limit of three posts. So the new added `rake servees` will create the spanish version with all the posts. And we need all the posts because the page shows three specific posts that happen not to be the three most recent posts

lpalma committed:
check that life_of_apprentice posts exist before including them

lpalma committed:
update EN Apprenticeship program description

lpalma committed:
update EN Craftsman benefits section

lpalma committed:
fixing identation for the Apprentice page

lpalma committed:
fix identation for the Become a Craftsman page

lpalma committed:
fix margin bottom and top for the apprentice page

lpalma committed:
fix structure of Life Of Apprentice blog posts

MiyamotoAkira committed:
Moved apply buttons

All buttons have been centered. The second button has been moved inside the div for the benefits (at the end)

MiyamotoAkira committed:
Changed the button for become_a_craftsman page

lpalma committed:
add padding to Craftsman page benefits section

lpalma committed:
increase margin between application button and role description

MiyamotoAkira committed:
Fixed regression of text

MiyamotoAkira committed:
Reformatted apply form

The form has been redone following the mockup and fields provided by David. Still some styling to be done and behaviour hasn't been changed yet

MiyamotoAkira committed:
Corrected indentation
